OpenRA.Game/Actor.cs

```diff
@@ -22,7 +22,7 @@
 
 namespace OpenRA
 {
-	public sealed class Actor : IScriptBindable, IScriptNotifyBind, ILuaTableBinding, ILuaEqualityBinding, ILuaToStringBinding, IEquatable<Actor>, IDisposable
+	public sealed class Actor : IScriptBindable, IScriptNotifyBind, ILuaTableBinding, ILuaEqualityBinding, ILuaToStringBinding, ISync, IEquatable<Actor>, IDisposable
```

Comment:
We have a special-case path for syncing actors, but that only includes the actor ID.